<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurvedMPRExtensionBase</class>
 <widget class="QWidget" name="CurvedMPRExtensionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_scrollToolButton">
           <property name="text">
            <string>Slicing</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/slicing.png</normaloff>:/images/slicing.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_zoomToolButton">
           <property name="text">
            <string>Zoom</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/zoom.png</normaloff>:/images/zoom.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_distanceToolButton">
           <property name="text">
            <string>Distance</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/distance.png</normaloff>:/images/distance.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_polylineROIToolButton">
           <property name="text">
            <string>ROI</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/polyline.png</normaloff>:/images/polyline.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_eraserToolButton">
           <property name="text">
            <string>Rubber</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/eraser2.png</normaloff>:/images/eraser2.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_voxelInformationToolButton">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../../main/main.qrc">
             <normaloff>:/images/voxelInformation.png</normaloff>:/images/voxelInformation.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>261</width>
         <height>26</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>WW/WL:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="udg::QWindowLevelComboBox" name="m_windowLevelComboBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="udg::ViewersLayout" name="m_viewersLayout" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>udg::QWindowLevelComboBox</class>
   <extends>QComboBox</extends>
   <header>qwindowlevelcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>udg::ViewersLayout</class>
   <extends>QWidget</extends>
   <header>viewerslayout.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="curvedmpr.qrc"/>
  <include location="../../../main/main.qrc"/>
  <include location="../../../main/main.qrc"/>
 </resources>
 <connections/>
</ui>
